volatility/framework/plugins/windows/envars.py

```diff
@@ -0,0 +1,161 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+# Creator: Aviel Zohar (memoryforensicsanalysis@gmail.com) # Fix vol2 plugin to vol3
+
+from typing import Callable, List, Generator, Iterable, Dict
+from volatility.framework import renderers, interfaces, objects, exceptions
+from volatility.framework.configuration import requirements
+from volatility.framework.objects import utility
+from volatility.plugins.windows import pslist
+from volatility.framework.symbols.windows.extensions.registry import RegValueTypes

```

Comment:
We try hard not to import types directly, so that people don't then import them from this file, but always know which module they belong.  We make a special case for typing because it's used so frequently, but otherwise please only import other modules.